exp/containers.md

```diff
@@ -63,6 +63,13 @@ Status            |  Description
 `deleted` | The container is deleted and will be removed from the AMS database soon.
 `error` | An error occurred while processing the container. The container is stopped. Further information about the error can be viewed with `amc show <container id>`.
 
+## Development mode
+
+AMS allows to start a container in a development mode. This will turn off certain features, which are active in a container otherwise. The mode is mainly useful for development of addons inside an Anbox container.
```

Comment:
```suggestion
AMS allows to start a container in development mode. This mode turns off some features that are usually active in a container. It is mainly useful when developing addons inside a container.
```
I was confused by the "Anbox container" here - but that's what we're talking about everywhere, right?